### PR TITLE
Add optimisim network

### PR DESCRIPTION
**Pull request by akremstudy**
This adds optimism and arbitrum to the directory plus a gas api source for both. The gas api response is different than the other api sources so had to do a work around for now.  Will probably need to be handled better for easier integration process as far as gas api sources go.

### CODE REVIEW

**Comment by akremstudy** [comment]
@oraclown, is this cool for now?

File: src/telliot_core/gas/legacy_gas.py [hunk 1]
```diff
@@ -86,13 +95,34 @@ async def legacy_gas_station(chain_id: int, speed: Optional[ethgastypes] = None,
...
             return None
 
-    if speed not in prices:
-        logger.error(f"Invalid gas price speed for gasstation: {speed}")
-        return None
-
-    if prices[speed] is None:
-        logger.error("Unable to fetch gas price from gasstation")
-        return None
+    if isinstance(speed, tuple):
```
Comment (oraclown) [inline src/telliot_core/gas/legacy_gas.py:98]: I think we can refactor this logic of retrieving the price like in #358 , as an example

**Review by oraclown — CHANGES_REQUESTED** [review]
thanks @akremstudy ! I will take a look at the failing code fmt stuff tomorrow. I think we should refactor this a little b4 merging. If @spuddyminer needs this to report ASAP, then switch to this branch and update the telliot-feeds dep to be this branch.. in a separate env.

**Comment by spuddyminer** [comment]
tested for reporting on optimism, and gas estimation works great!

**Review by oraclown — APPROVED** [review]
Nice, thanks @akremstudy !

**Pull request merged**